__tests__/app.test.js

```diff
@@ -136,9 +136,52 @@ describe('/GET /api/articles', () => {
         expect(article.body).hasOwnProperty('article_img_url');
         expect(article.body).not.hasOwnProperty('body')
         expect(article.body).hasOwnProperty('number_of_comments')
-        console.log(article.number_of_comments)
         expect(typeof article.number_of_comments).toBe('string')
       })
     })
   })
 })
+
+describe('/api/articles/:article_id/comments', () => {
+  test('GET /api/articles/:article_id/comments responds with an array of comments for the given article_id ', () => {
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then((res) => {
+      expect(Array.isArray(res.body)).toBe(true)
+      expect(res.body.length).not.toBe(0)
```

Comment:
We could be more specific here and say exactly what we expect the length to be, remembering that we have access to the test data and we know that's what we're seeding and then testing against.